### PR TITLE
TypeScript: fix namespace comment printing

### PR DESCRIPTION
**Pull request by azz**
Comments weren't printed between TSModuleDeclaration and TSModuleBlock.

#1643

### CODE REVIEW

File: tests/typescript/compiler/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -494,3 +526,309 @@ interface I {
...
+      import m5_nonerrorImport = glo_M1_public;
+    }
+  }
+}
+
+declare module "anotherParseError" {
+  namespace m2 {
+    //declare module "abc" {
+    //}
+
```
Comment (vjeux) [inline tests/typescript/compiler/__snapshots__/jsfmt.spec.js.snap]: there is an extra \n after the comment

File: src/printer.js [hunk 2]
```diff
@@ -2472,6 +2472,10 @@ function genericPrintNoParens(path, options, print, args) {
       } else if (n.body) {
         parts.push(
           " {",
+          path.call(
+            bodyPath => comments.printDanglingComments(bodyPath, options),
+            "body"
+          ),
           indent(concat([line, group(path.call(print, "body"))])),
           line,
```
Comment (vjeux) [inline src/printer.js]: there are two `line` that are always printed. Probably want to make one only when n.body exists.

File: tests/typescript/compiler/__snapshots__/jsfmt.spec.js.snap (hunk 1 above)
Comment (azz) [inline tests/typescript/compiler/__snapshots__/jsfmt.spec.js.snap]: Fixed.

File: src/printer.js (hunk 2 above)
Comment (azz) [inline src/printer.js]: Just pushed 66c10372ecc75c4ffae590948da1d016ba97a0ed, is that correct?

Comment (vjeux) [inline src/printer.js]: That seems to fix it!

File: src/printer.js [hunk 3]
```diff
@@ -2472,7 +2472,15 @@ function genericPrintNoParens(path, options, print, args) {
       } else if (n.body) {
         parts.push(
           " {",
-          indent(concat([line, group(path.call(print, "body"))])),
+          indent(concat([
```
Comment (vjeux) [inline src/printer.js]: this is going to fail because of the prettier lint rule. You now need to do `yarn lint -- --fix` before committing :p

Comment (azz) [inline src/printer.js]: Whoops! Need to figure out how to get format-on-save working in VSCode. It keeps looking for prettier in `node_modules` :P

**Pull request merged**